android/app/build.gradle

```diff
@@ -35,8 +35,8 @@ android {
     ndkVersion flutter.ndkVersion
 
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
+        sourceCompatibility JavaVersion.VERSION_17
+        targetCompatibility JavaVersion.VERSION_17
```

Comment:
Why did you change it?

Reply:
Probably this was needed  because of some demands of Android Studio. My advice is to make such changes only locally (at least for now, we can discuss global change of java version ) and don't push them on github. 

Please change them back to 1.8